docs/website/docs/overview/installation.md

```diff
@@ -29,12 +33,12 @@ Installing `odo` on `amd64` architecture:
 
 1. Download the latest release from the mirror:
 ```shell
-curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.0.0/odo-linux-amd64 -o odo
+curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.3.0/odo-linux-amd64 -o odo
```

Comment:
I agree it will reduce the maintenance overhead for us. But I also think it is generally better to be explicit with versions if we don't want to end up with a similar situation during the v2-v3 transition.
Even from a user perspective, I think it helps clarify what they are downloading and installing, no?

Reply:
I even think some automation can help reduce that maintenance burden.
I was thinking a few weeks ago of automating this task with a GitHub Workflow (triggered on a new tag push). I find it too repetitive and too manual :)

This Workflow would create a draft PR that:
1. Generates the release notes and adds them as a draft release blog post
2. Updates the `build/VERSION` file
3. Updates this installation page with the new version

Reply:
I do like that idea. Let's do that.